src/tests/unit/tests/injected/adapters/resolution-creator.test.ts

```diff
@@ -62,8 +62,6 @@ describe('ResolutionCreator', () => {
                 ],
                 all: [{ id: null, message: 'all 1 message', data: null }],
                 none: [{ id: null, message: 'none 1 message', data: null }],
-                html: 'test html',
```

Comment:
why this is removed?

Reply:
NodeResolutionData Inteface does not consist of html property, checked git history. Thus, removed the property.